setup.py

```diff
@@ -13,7 +13,7 @@
     url="https://github.com/microsoft/playwright-pytest",
     packages=["pytest_playwright"],
     include_package_data=True,
-    install_requires=["playwright==1.8.0a1", "pytest", "pytest-base-url"],
+    install_requires=["playwright==1.9.0", "pytest", "pytest-base-url"],
```

Comment:
@mxschmitt there wasn't any breaking [changes](https://github.com/microsoft/playwright-python/releases/tag/v1.9.0) so 
```suggestion
    install_requires=["playwright>=1.9.0", "pytest", "pytest-base-url"],
```